src/javascript/core/store.js

```diff
@@ -152,6 +110,29 @@ const Store = function (name, config) {
 		}
 	}
 
+	// If the user has previous data stored in the old cookie storage system
+	// import the data into the new storage system and remove them from the cookie.
+	const oldCookieStoreData = cookieLoad(this.storageKey);
+	if (oldCookieStoreData) {
+		try {
+			const data = JSON.parse(oldCookieStoreData);
+			if (this.data) {
+				Object.assign(this.data, data);
```

Comment:
Do we want keys in the cookie to replace keys in local storage?

Reply:
Good question, I was thinking we did, yes. It's not often that both storage are used on the same device so I didn't think it would matter which one override the other.